src/vs/editor/test/common/mocks/testConfiguration.ts

```diff
@@ -47,4 +47,8 @@ export class TestConfiguration extends CommonEditorConfiguration {
 			maxDigitWidth: 10,
 		}, true);
 	}
+
+	public reserveHeight(height: number): void {
```

Comment:
This will no longer be necessary if we move it to `/common/` as mentioned above.